packages/dds/merge-tree/src/test/client.rollbackFarm.spec.ts

```diff
@@ -40,7 +40,7 @@ describe("MergeTree.Client", () => {
                 mt.seedWithArray([0xDEADBEEF, 0xFEEDBED, minLength, opsPerRollback]);
 
                 // A: readonly, B: rollback, C: rollback + edit, D: edit
-                const clients = createClientsAtInitialState({ initialState: "" }, "A", "B", "C", "D");
+                const clients = createClientsAtInitialState({ initialState: "", options: {mergeTreeUseNewLengthCalculations: true} }, "A", "B", "C", "D");
```

Comment:
I originally forgot to take this out as this was one thing I tried as a fix, but would it be beneficial to leave it?

Reply:
It looks like there will be an effect for this setting, but I'd err on the side of keeping the default to match what customers will likely have. So remove it if we don't absolutely need it.